src/main/java/bio/terra/app/controller/CohortBuilderApiController.java

```diff
@@ -0,0 +1,432 @@
+import bio.terra.model.DisplayHint;
+import bio.terra.model.DisplayHintDisplayHint;
+import bio.terra.model.DisplayHintEnum;
+import bio.terra.model.DisplayHintEnumEnumHintValues;
+import bio.terra.model.DisplayHintList;
+import bio.terra.model.DisplayHintNumericRange;
+import bio.terra.model.HintQuery;
+import bio.terra.model.Instance;
+import bio.terra.model.InstanceCount;
+import bio.terra.model.InstanceCountList;
+import bio.terra.model.InstanceHierarchyFields;
+import bio.terra.model.InstanceList;
+import bio.terra.model.InstanceRelationshipFields;
+import bio.terra.model.Literal;
+import bio.terra.model.LiteralValueUnion;
+import bio.terra.model.Query;
+import bio.terra.model.ValueDisplay;
+import io.swagger.annotations.Api;
+import java.time.OffsetDateTime;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+@Api(tags = {"cohort-builder"})
+public class CohortBuilderApiController implements CohortBuilderApi {
+
+  @Autowired
+  public CohortBuilderApiController() {}
+
+  @Override
+  public ResponseEntity<CohortList> listCohorts(
+      @RequestParam(value = "offset", required = false, defaultValue = "0") Integer offset,
+      @RequestParam(value = "limit", required = false, defaultValue = "10") Integer limit) {
+    CohortList response = new CohortList();
+    response.add(generateStubCohort());
+    return new ResponseEntity<>(response, HttpStatus.OK);
+  }
+
+  @Override
+  public ResponseEntity<Cohort> createCohort(CohortCreateInfo cohortCreateInfo) {
+    return new ResponseEntity<>(generateStubCohort(), HttpStatus.OK);
+  }
+
+  @Override
+  public ResponseEntity<Cohort> getCohort(String cohortId) {
+    Cohort response = generateStubCohort();
+    return new ResponseEntity<>(response, HttpStatus.OK);
+  }
+
+  @Override
+  public ResponseEntity<Cohort> updateCohort(String cohortId, CohortUpdateInfo cohortUpdateInfo) {
+    return new ResponseEntity<>(generateStubCohort(), HttpStatus.OK);
+  }
+
+  @Override
+  public ResponseEntity<Void> deleteCohort(String cohortId) {
+    return new ResponseEntity<>(HttpStatus.OK);
+  }
+
+  @Override
+  public ResponseEntity<InstanceCountList> countInstances(
+      String entityName, CountQuery countQuery) {
+    // Stub values based on what the API synthetic data returns in the other API - not
+    // representative of life
+    List<ValueDisplay> yearsOfBirth = new ArrayList<>();
+    for (long i = 1909; i <= 1983; i++) {
+      yearsOfBirth.add(
+          new ValueDisplay()
+              .value(
+                  new Literal()
+                      .dataType(DataType.INT64)
+                      .valueUnion(new LiteralValueUnion().int64Val(i))));
+    }
```

Comment:
This is fine since it's all temporary code anyway, but in the future you can write forms like this using streaming
```suggestion
    List<ValueDisplay> yearsOfBirth = LongStream.range(1909, 1984).mapToObj(i ->
          new ValueDisplay()
              .value(
                  new Literal()
                      .dataType(DataType.INT64)
                      .valueUnion(new LiteralValueUnion().int64Val(i))));
```